bot/plugins/util.py

```diff
@@ -18,6 +18,10 @@
     f" `{config.PREFIX}asm` message prefix."
 )
 
+REPO_LINK = "https://github.com/Lunarmagpie/io"
+
+INVITE_URL = "https://discord.com/api/oauth2/authorize?client_id=1073771658906701954&permissions=346176&scope=bot"
```

Comment:
can you move these to `config.py.example`? don't use from when you import config.